ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.18__4.8.yaml

```diff
@@ -0,0 +1,208 @@
+base_images:
+  base:
+    name: "4.8"
+    namespace: ocp
+    tag: base
+  cli:
+    name: "4.8"
+    namespace: ocp
+    tag: cli
+  openshift_knative-v0.23.0_knative-eventing-src:
+    name: knative-v0.23.0
+    namespace: openshift
+    tag: knative-eventing-src
+  openshift_knative-v0.23.1_knative-serving-src:
+    name: knative-v0.23.1
+    namespace: openshift
+    tag: knative-serving-src
+  openshift_knative-v0.23.5_knative-eventing-kafka-src:
+    name: knative-v0.23.5
+    namespace: openshift
+    tag: knative-eventing-kafka-src
+  openshift_release_golang-1.15:
+    name: release
+    namespace: openshift
+    tag: golang-1.15
+build_root:
+  project_image:
+    dockerfile_path: openshift/ci-operator/build-image/Dockerfile
+images:
+- context_dir: openshift/ci-operator/source-image
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+    openshift_knative-v0.23.0_knative-eventing-src:
+      as:
+      - registry.ci.openshift.org/openshift/knative-v0.23.0:knative-eventing-src
+    openshift_knative-v0.23.1_knative-serving-src:
+      as:
+      - registry.ci.openshift.org/openshift/knative-v0.23.1:knative-serving-src
+    openshift_knative-v0.23.5_knative-eventing-kafka-src:
+      as:
+      - registry.ci.openshift.org/openshift/knative-v0.23.5:knative-eventing-kafka-src
+  to: serverless-operator-src
+- context_dir: .
+  dockerfile_path: serving/ingress/Dockerfile
+  from: base
+  inputs:
+    openshift_release_golang-1.15:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.15
+  to: knative-openshift-ingress
+- context_dir: .
+  dockerfile_path: knative-operator/Dockerfile
+  from: base
+  inputs:
+    openshift_release_golang-1.15:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.15
+  to: knative-operator
+- context_dir: .
+  dockerfile_path: openshift-knative-operator/Dockerfile
+  from: base
+  inputs:
+    openshift_release_golang-1.15:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.15
+  to: openshift-knative-operator
+- context_dir: olm-catalog/serverless-operator
+  from: base
+  to: serverless-bundle
+- context_dir: olm-catalog/serverless-operator
+  dockerfile_path: stopbundle.Dockerfile
+  from: base
+  to: serverless-stop-bundle
+promotion:
+  name: openshift-serverless-v1.18.0
+  namespace: openshift
+releases:
+  latest:
+    release:
+      channel: stable
+      version: "4.8"
+resources:
+  '*':
+    limits:
+      memory: 6Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: operator-e2e-aws-ocp-48
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws
```

Comment:
You don't want cluster pools for 4.8 ?

Reply:
Alright. It's not necessary. I'm just asking if this was intentional or you'd want cluster pools. I can mark this as lgtm